controls/listview/properties-methods-events.md

```diff
@@ -8,7 +8,11 @@ published: True
 position: 10 
 ---
 
-# Properties
```

Comment:
Always include some kind of introduction to every article. Don't rely solely on the title.